package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -188,12 +188,13 @@ def __init__(self, filename, **kwargs):
         self.n_atoms = kwargs.pop('n_atoms', None)
         if self.n_atoms is None:  # this should be done by parsing DATA first
             raise ValueError("DATAReader requires n_atoms keyword")
+        self.atom_style = kwargs.pop('atom_style', None)
```

Comment:
`atom_style` is not documented anywhere in the user-accessible docs – or at least I don't see it.